plugins/dev_tools/commands/rename_command.py

```diff
@@ -34,12 +36,12 @@ def rename_command(ctx, group_name: list[str], old_command_name: str, new_comman
     group_name = [name.lower().replace("-", "_") for name in group_name]
     old_command_name = old_command_name.lower().replace("-", "_")
     new_command_name = new_command_name.lower().replace("-", "_")
-    babylon_groups_path = BABYLON_PATH / "groups"
```

Comment:
base_dir is passed in context with the option `--plugin` to allow the command to affect a plugin directory instead of Babylon, if option is not set the value of `base_dir` is set to `BABYLON_PATH` but if the option is set we set the option to the path of the plugin instead

Reply:
https://github.com/Cosmo-Tech/Babylon/blob/6e3f7dcc0efa31d0ddddfa1c1c267d67b46145bf/plugins/dev_tools/__init__.py#L24